deploy/zfs-operator.yaml

```diff
@@ -657,7 +657,7 @@ spec:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
         - name: openebs-zfs-plugin
-          image: quay.io/openebs/zfs-driver:ci
+          image: quay.io/openebs/zfs-driver:v0.6
```

Comment:
To comply with other openebs naming conventions, make the tag as `0.6.0`
(https://github.com/openebs/zfs-localpv/releases/tag/0.6.0)

Reply:
done.